### ISSUE TITLE
[Bug]: Cannot specify CHECK() clause content in the Constraints tab

### ISSUE THREAD

**Issue by foxyseta** [sqlitebrowser/sqlitebrowser#3252]
### What did you do?

1. open the _Edit Table Definition_ window
2. open the _Constraints_ tab
3. open the _Add Constraint_ dropdown menu
4. select the _Check_ option
5. look at the row representing the newly created check constraint

### What did you expect to see?

A column in the Constraints table for inserting the SQL to be put inside the CHECK clause.
When I create Check constraints inlined within a field declaration, such columns exists (see the _Fields_ tab in the _Edit Table Definition_ window).

### What did you see instead?

No such column.

### DB4S Version

3.12.99 (nightly)

### What OS are you seeing the problem on?

Linux

### OS version

Fedora release 37 (Thirty Seven)

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by chrisjlocke** [comment]
Can you give me an example of what you're trying to check?  Eg, the value is 4, or above 9?
Also, can you click 'Help' and 'About' and specify the date in brackets?

![image](https://user-images.githubusercontent.com/7872218/210053791-4f1d667e-c3ee-464d-90f5-0aa4c02204e9.png)

3.12.99 designates the version is a nightly release, but the date of that nightly would be handy.
While this bug/issue probably hasn't been resolved lately, grabbing the latest source and building from that solves some other furballs in Linux.

**Comment by foxyseta** [comment]
> Can you give me an example of what you're trying to check? Eg, the value is 4, or above 9?

Sure! I'd like to check for a given field to be less than a second field. Because multiple table fields are involved, I'd like this condition to be specified as a table constraint rather than be attached to a single column definition. In other words, I'd like this SQL to be generated:

```sql
CREATE TABLE "MyTable" (
	"A"	INTEGER,
	"B"	INTEGER,
	CHECK ("A" < "B")
);
```
Rather than:

```sql
CREATE TABLE "MyTable" (
	"A"	INTEGER,
	"B"	INTEGER CHECK ("A" < "B"),
);
```

> Also, can you click 'Help' and 'About' and specify the date in brackets?

The date is `Jul 23 2022`.

**Comment by chrisjlocke** [comment]
So in the 'Modify table' dialog, if you click the 'Check constraints' tab, you can just fill the values.

![image](https://user-images.githubusercontent.com/7872218/210089371-1c8e2c87-bc8d-4e06-97a1-2a581003e927.png)

I've two entries as I did one manually, then used the dialog on the second, which produced the same result. :)

**Comment by foxyseta** [comment]
> So in the 'Modify table' dialog, if you click the 'Check constraints' tab, you can just fill the values.

At the risk of sounding extremely silly, I only see two tabs in the Edit table definition window. Besides the regular _Fields_ tab, there is a single tab for any kind of constraints (Indexes, Primary Keys, Foreign Keys, and Checks).

![image](https://user-images.githubusercontent.com/12249077/210102200-aa0a1373-a52d-4fd0-a763-b4f6afb980f9.png)

So yeah, my GUI looks very different from yours.

> (You don't need the quotation marks around the field names .. DB4S seems to like adding these to everything...)

It's true, literally everywhere 🙈

**Comment by chrisjlocke** [comment]
Interesting.  Your version is 23 Jul 2022, but that 'fix' for the constraint was on 24 Jul.  
If you build DB4S from the latest source, that should work.  
Alternatively, you can just slap the required SQL in a tab and run it.

**Comment by foxyseta** [comment]
If this is already fixed in the latest release (v3.12.2), I could also use its binaries, right?
